.github/workflows/task-test.yml

```diff
@@ -181,7 +157,5 @@ jobs:
           if-no-files-found: ignore
```

Comment:
fix path here as well

Reply:
right

Reply:
was it fixed in RLTest?

Reply:
This is the fix in RLest - https://github.com/RedisLabsModules/RLTest/pull/219